src/main/java/frc/robot/subsystems/BallShooter.java

```diff
@@ -97,40 +99,109 @@ public void periodic() {
             timer = timer + 20;
         }
     }
-     
+
     // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CMDPIDGETTERS
 
     // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CMDPIDGETTERS
     public void fireMotor() {
-        shootMotor.set(-.85);
+        toConsoleln("fireMotor() called");
```

Comment:
this is the driver's primary shoot mechanism -- think you want to fix verse disable